src/Lesson10/ex_30.java

```diff
@@ -0,0 +1,36 @@
+package Lesson10;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+//Создать коллекцию, наполнить ее случайными числами. Удалить повторяющиеся числа.
+public class ex_30 {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Enter the number of members (n):");
+        int n = sc.nextInt();
+        sc.close();
+        List<Integer> collection = new ArrayList<>(n);
+        for (int i = 0; i < n; i++) {
+            collection.add((int) (Math.random() * 10) - 5);
+        }
+        System.out.println("The first version of marks: ");
+        for (int i = 0; i < n; i++) {
+            System.out.print(collection.get(i) + "\t");
+        }
+        for (int i = 0; i < n; i++) {
```

Comment:
Better way here is to use wrappers (Integer instead of int) and filter with Set, but your solution is good too )